### PR TITLE
Init

### PR DESCRIPTION
**Pull request by goruha**
## What
* Added redis failover cluster based on aws elasticache

## Why
* Redis is a popular storage/cache backend that offers more advanced data structures than Memcache

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    CacheClusterId = "${module.label.id}"
+  }
+
+  alarm_actions = ["${var.alarm_actions}"]
+  depends_on    = ["aws_elasticache_replication_group.default"]
+}
+
+
+module "dns" {
+  source    = "git::https://github.com/cloudposse/tf_hostname.git"
```
Comment (osterman) [inline main.tf]: Lets tag & pin.

**Comment by osterman** [comment]
- [ ] Fix `LICENSE` copyright.
- [ ] Update `README.md` with usage

File: main.tf (hunk 1 above)
Comment (goruha) [inline main.tf]: Done

**Comment by goruha** [comment]
>  Fix LICENSE copyright.
What is this about?

**Comment by osterman** [comment]
https://github.com/cloudposse/tf_redis/blob/master/LICENSE#L189

File: README.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+
+  automatic_failover = "false"
+}
+```
+
+## Input
+
+|  Name  |  Default  |  Decription  |
+|:------:|:---------:|:------------:|
+| namespace|global|Namespace|
```
Comment (osterman) [inline README.md]: spacing?

**Pull request merged**